README.md

```diff
@@ -87,7 +87,7 @@ There are two ways to start a debug session:
    ![CMSIS view](https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/raw/main/images/cmsis-view-icon.png),
    click on the **Debug** icon
    ![Debug icon in the CMSIS view](https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/raw/main/images/debug-icon.png).
-   The current configurations in your `launch.json` file will be used to start the debug session.
+   The first configuration in your `launch.json` file will be used to start the debug session.
```

Comment:
The configuration for the debugger configured in the active `target-set` is written to the `launch.json` file and will be used to start the debug session.